CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log - Power BI Custom Visuals API
 
+# 5.1.0 
+* Added Subtotal position/early fetch API 
+* Added Custom Sorting API
+
+# 5.0.0
+* Add new formatting pane `FormattingModel` interfaces
+
+# 4.7.0
```

Comment:
what is this 'drill api'?

Reply:
Spoke with Ramiz and he said that this is exactly what he did in 4.7.0 and he ok with it